modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -553,6 +553,7 @@ the form:
         "OutputType" : "native",
         "Filename" : "abc.mnc",
         "AcquisitionType" : "t1w/t2w/etc",
+        "IsPhantom" : true/false,
```

Comment:
I was going to say this isn't valid js so it's going to be rendered as an error since it's in a JS block, but then I decided to check if it's valid js in my JS console..

```
>> $x = true / false
Infinity
```
....